### PR TITLE
Update F# coding conventions to clear up bad sentence

### LINKED ISSUE #5495: Incomplete sentence about open statements
The following sentence is not complete and does not make sense:

> In F#, because elements opened into a scope can shadow others already present.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 402e43bd-76b8-a1c8-2c62-e14017f5ed1e
* Version Independent ID: 990e56cd-648d-45c4-30e0-b221874ec329
* Content: [F# coding conventions](https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/conventions#feedback)
* Content Source: [docs/fsharp/style-guide/conventions.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/style-guide/conventions.md)
* Service: **unspecified**
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #5495

### CODE REVIEW

File: docs/fsharp/style-guide/conventions.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ let parsed = StringTokenization.parse s // Must qualify to use 'parse'
 
 In F#, the order of declarations matters, including with `open` statements. This is unlike C#, where the effect of `using` and `using static` is independent of the ordering of those statements in a file.
 
-In F#, because elements opened into a scope can shadow others already present. This means that reordering `open` statements can alter the meaning of code. As a result, sorting alphanumerically (or pseudorandomly) is generally not recommended, lest you generate different behavior that you might expect.
+In F#, elements opened into a scope can shadow others already present. This means that reordering `open` statements could alter the meaning of code. As a result, sorting all `open` statements alphanumerically (or pseudorandomly) is generally not recommended, lest you generate different behavior that you might expect.
```
Comment (svick) [inline docs/fsharp/style-guide/conventions.md]: Why does it make sense to mention pseudorandom sorting of `open` statements here? Is that something someone would actually consider?

Comment (cartermp) [reply docs/fsharp/style-guide/conventions.md]: That's fair, the issue here is _any_ sorting which isn't topological is likely to result in unwanted shadowing.

**Review by BillWagner — APPROVED** [review]
This LGTM @cartermp 

You can :shipit: when ready.

**Comment by cmeeren** [comment]
Isn't `alphanumericall` is spelled with a single `l`?

**Comment by cmeeren** [comment]
Or rather, `alphanumerically` is probably the correct word in that context.

**Comment by mairaw** [comment]
@cmeeren I've opened PR #5526 to fix this.

**Pull request merged**